download/finder.go

```diff
@@ -6,19 +6,32 @@ import (
 	"net/http"
 	"os"
 	"path/filepath"
+	"regexp"
 	"strings"
+	"time"
 
 	"github.com/PuerkitoBio/goquery"
 )
 
 const federalRevenue = "https://www.gov.br/receitafederal/pt-br/assuntos/orientacao-tributaria/cadastros/consultas/dados-publicos-cnpj"
 
+var reUpateDate = regexp.MustCompile(`(?i)data da última extração:.*([0-9]{2}/[0-9]{2}/[0-9]{4})`)
```

Comment:
Usar `\d` para dígitos e `\s` para espaços acho que torna essa expressão regular mais precisa e mais fácil de ler:

```suggestion
var reUpateDate = regexp.MustCompile(`(?i)data da última extração:\s*(\d{2}/\d{2}/\d{4})`)
```

Reply:
Good point! 👍